src/NuGet.LibraryModel/LibraryDependency.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Text;
+
+namespace NuGet.LibraryModel
+{
+    public class LibraryDependency
+    {
+        public LibraryRange Range { get; }
+
+        public LibraryDependencyType Type { get; } = LibraryDependencyType.Default;
+
+        public LibraryIdentity ResolvedIdentity { get; set; }
+
+        public string Name
+        {
+            get
+            {
+                return Range.Name;
+            }
```

Comment:
Simple getter like this could be put on one line to save some screen real estate. Reader would love to have more information in one screen.